File: app/src/main/java/com/stevenfrew/beatprompter/PauseOnScrollStartMessage.java
package com.stevenfrew.beatprompter;

class PauseOnScrollStartMessage extends SignalOnlyBluetoothMessage
{
    static final byte PAUSE_ON_SCROLL_START_MESSAGE_ID=3;

    PauseOnScrollStartMessage()
    {
        super(PAUSE_ON_SCROLL_START_MESSAGE_ID);
    }

    PauseOnScrollStartMessage(byte[] bytes) throws NotEnoughBluetoothDataException
    {
        super(bytes);
    }
}

File: app/src/main/kotlin/com/stevenfrew/beatprompter/bluetooth/message/PauseOnScrollStartMessage.kt
package com.stevenfrew.beatprompter.bluetooth.message

/**
 * Bluetooth message that instructs the receiver to pause, because the band leader is scrolling
 * the display.
 */
class PauseOnScrollStartMessage : SignalOnlyBluetoothMessage(PAUSE_ON_SCROLL_START_MESSAGE_ID) {
    companion object {
        internal const val PAUSE_ON_SCROLL_START_MESSAGE_ID: Byte = 3
    }
}
